src/benchmarklib/benchmark_sql_executor.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "sql/sql_pipeline.hpp"
+#include "utils/sqlite_wrapper.hpp"
+
+namespace opossum {
+
+class TransactionContext;
+
+// This class provides SQL functionality to BenchmarkItemRunners. See AbstractBenchmarkItemRunner:_execute_item.
+class BenchmarkSQLExecutor {
+ public:
+  // If sqlite_wrapper is set, all SQL queries executed in the benchmark will also be executed using SQLite and then
+  // validated.
+  BenchmarkSQLExecutor(bool enable_jit, const std::shared_ptr<SQLiteWrapper>& sqlite_wrapper,
+                       const std::optional<std::string>& visualize_prefix);
```

Comment:
Please doc "visualize_prefix", I'm not perfectly sure what path (?) it contains

Reply:
Like this?

Reply:
if you'd like to make me very happy, also doc (here) what the suffix is comprised of

Reply:
That's the job of the caller and there is no real methodology. For TPC-H, it has a query number, for TPC-C, it is the name of the procedure, ...

Reply:
well, that's not true, right? The suffix is just "LQP/PQP" + index. Will doc that